components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -60,9 +61,26 @@ namespace scream
 class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess>
 {
 public:
-  using TimeStamp       = util::TimeStamp;
-  using ci_string       = ekat::CaseInsensitiveString;
-  using ci_string_pair  = std::pair<ci_string,ci_string>; // Group and grid names
+  using TimeStamp      = util::TimeStamp;
+  using ci_string      = ekat::CaseInsensitiveString;
+
+  // A group request allow to request a group of fields.
```

Comment:
Worth mentioning how GroupRequests are used? Okay if not--I'm just wondering if there's an easy example.

Reply:
Right, we don't have a stand alone example, only the usage of GroupRequest's in shoc and homme. I'll add some more comments here.